warehouse/views.py

```diff
@@ -322,8 +324,23 @@ def list_classifiers(request):
     has_translations=True,
 )
 def search(request):
+    ratelimiter = request.find_service(IRateLimiter, name="search", context=None)
     metrics = request.find_service(IMetricsService, context=None)
 
+    ratelimiter.hit(request.remote_addr)
+    if not ratelimiter.test(request.remote_addr):
+        metrics.increment("warehouse.search.ratelimiter.exceeded", tags=[])
```

Comment:
nit: do we need to expressly add tags as an empty list (here and in .hit below), or were we expecting some tags?

Reply:
that's good old fashioned copy-pasta, removed in a4b751cc1